assets/airports/tncm.json

```diff
@@ -9,433 +10,3385 @@
+                "GOUDA": ["GOUDA", "ILORA"],
+                "MNOLO": ["MNOLO", "_ULUBA332007.4", "_ULUBA326007.5", "_ULUBA320007.5", "_ULUBA313007.4", "_ULUBA307007.1", "_ULUBA301006.7", "_ULUBA295006.2", "_ULUBA290005.6", "_ULUBA285005", "_ULUBA280004.2", "_ULUBA278003.4", "_ULUBA276002.3"],
+                "SLUGO": ["SLUGO", "_PJM276SLUGO160"],
+                "TRNKY": ["TRNKY", "_ULUBA320007.5", "_ULUBA313007.4", "_ULUBA307007.1", "_ULUBA301006.7", "_ULUBA295006.2", "_ULUBA290005.6", "_ULUBA285005", "_ULUBA280004.2", "_ULUBA278003.4", "_ULUBA276002.3"],
+                "ZPATA": ["ZPATA", "_ULUBA258008.3", "_ULUBA262008.5", "_ULUBA267008.5", "_ULUBA271008.1", "_ULUBA274007.5", "_ULUBA276007"]
+            },
+            "body": ["ULUBA", "PJM"],
+            "rwy": {
+                "TNCM10": []
+            },
+            "draw": [
+                ["ULUBA*"],
+                ["GOUDA*", "ILORA", "ULUBA", "PJM"],
+                ["DANDE*", "_PJM276DANDE040"],
+                ["ELOPO*", "_ULUBA238006", "_ULUBA243006.1", "_ULUBA248006.1", "_ULUBA253006", "_ULUBA257005.8", "_ULUBA262005.5", "_ULUBA266005.2", "_ULUBA270004.8", "_ULUBA272004.3", "_ULUBA274003.8", "_ULUBA276002.3"],
+                ["MNOLO*", "_ULUBA332007.4", "_ULUBA326007.5", "_ULUBA320007.5", "_ULUBA313007.4", "_ULUBA307007.1", "_ULUBA301006.7", "_ULUBA295006.2", "_ULUBA290005.6", "_ULUBA285005", "_ULUBA280004.2", "_ULUBA278003.4", "_ULUBA276002.3"],
+                ["SLUGO*", "_PJM276SLUGO160"],
+                ["TRNKY*", "_ULUBA320007.5"],
+                ["ZPATA*", "_ULUBA258008.3", "_ULUBA262008.5", "_ULUBA267008.5", "_ULUBA271008.1", "_ULUBA274007.5", "_ULUBA276007"]
+            ]
         }
     },
     "spawnPatterns":[
         {
-            "origin": "tncm",
+            "origin": "TNCM",
+            "destination": "",
+            "category": "departure",
+            "route": "TNCM10.MODOR2.JUICE.B520.STT",
+            "altitude": "",
+            "speed": "",
+            "method": "random",
+            "rate": 1.56,
+            "airlines": [
+                ["aal/tncmLong", 6],
+                ["aal/tncmShort", 4],
+                ["bwa/b738", 2],
+                ["cey/crj2", 4],
+                ["dal/b73x", 4],
+                ["eja", 4],
+                ["n/fastGA", 1],
+                ["pj/fastGA", 1],
+                ["swg", 2],
+                ["wja/b73x", 2]
+            ]
+        },
+        {
+            "origin": "TNCM",
+            "destination": "",
+            "category": "departure",
+            "route": "TNCM10.BOPAT2.JUICE.B520.STT",
+            "altitude": "",
+            "speed": "",
+            "method": "random",
+            "rate": 3.34,
+            "airlines": [
+                ["fdx/feeder", 3],
+                ["guy/at45", 3],
+                ["iwy", 3],
+                ["lia", 4],
+                ["n/lightGA", 1],
```

Comment:
this fleet contains a reference to a BE36. There is a file for this, but the icao property of that file is BE20 (surely a copy-paste error). Could you verify the details are right for the BE36 (or adjust as necessary) and then rename the icao property to match?

Reply:
I have no idea how long that's been like that 😬